report.php

```diff
@@ -138,14 +122,32 @@
 $where = implode(' AND ', $where);
 
 $sql = "SELECT x.id, x.eventname, u.firstname, u.lastname, x.contextid, x.timecreated, $extraselect
-          FROM {$basetable} x
+          FROM {{$basetable}} x
```

Comment:
Is this a typo? {{$basetable}}

Reply:
Nope one for concat, one for moodle table handler.